test/DynamoCoreTests/IconsTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Dynamo.Models;
+using Dynamo.Search.SearchElements;
+using Dynamo.UI;
+using Dynamo.Wpf.Services;
+using DynamoShapeManager;
+
+using NUnit.Framework;
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Windows.Media;
+
+namespace Dynamo.Tests
+{
+    [TestFixture]
+    internal class IconsTests
+    {
+        private static string geometryFactoryPath = "";
+        private IEnumerable searchEntries;
+        private IconServices iconServices = new IconServices();
```

Comment:
I think `IconServices` can only be instantiated with an `IPathManager` now, you can get it from `DynamoModel.PathManager` property after `DynamoModel` is created.